### DISCUSSION TITLE
New shortcut

### DISCUSSION THREAD

**Discussion by bbfreed** [arkypita/LaserGRBL#2260] [Ideas]
Hello,
I burn wooden tomato stakes for my darling (about 300 pieces)

I often use laserGRBL for the same picture.  (I change the picture but keep the same name)

So, I use the Alt+F and Alt+R to reload the last file.

But for the the import raster image windows and target image windows, the next and create buttons have no shortcut.

Do you think it's possible to assign one for this two buttons ?

Best regards

And thank you for your work !

**Comment by Tribo-2dot0** [comment]
No menu Grbl > Hotkeys você pode criar seus próprios atalhos em CustomButton (10 no total)

**Reply by bbfreed** [reply]
Hello,

Thank you fot the answer. is there's a way to do it with G-code ?

**Reply by Tribo-2dot0** [reply]
O Gcode contido em Custom Buttons?
São justamente esses que você pode personalizar, os três primeiros não contam... a partir do quarto botão você Ctrl+1, Ctrl+2, etc.
Você pode criar um botão com GCode e colocar na ordem para o atalho.

**Reply by Tribo-2dot0** [reply]
Agora que vi que você quer atalhos para os botões Next e Create... desconheço se há uma maneira de fazer isso.

**Reply by bbfreed** [reply]
In fact, i don't really need a hotkey, maybe only if the next and create buttons can have the focus or have it after use of few tabulation key, i will don't have to use the mouse anymor and it 's more faster to reload the file.

**Reply by bbfreed** [reply]
I'll try to explain my idea differently. Is it possible to have the focus on create and next buttons.

I just want to type Ctrl+R and the enter's key 2 times to be ready for a new burning.

![image](https://github.com/user-attachments/assets/1eb4e8e1-90cd-4c25-bee8-0576308b45af)
